src/components/RoundImageRow/RoundImageRow.scss

```diff
@@ -1,15 +1,31 @@
-.plus {
+.plus-default {
   position: relative;
   width: $space-9x;
   height: $space-9x;
   line-height: $space-7x;
-  margin-top: -$space-4x;
   border-radius: 50%;
   border: 3px solid $color-white;
   composes: caption-lt-lg from 'css/typography.scss';
+  text-align: center;
+  margin-top: -$space-4x;
   background: $color-caribbean-green;
   color: $color-white;
+}
+
+.plus-profile {
+  position: relative;
```

Comment:
looks like a lot of duplicate stuff from the one above, should put the common stuff in a shared class. also, i wonder if the additional/changed styles are better handled by passing in a classname to the component

Reply:
I tried doing this:

```
%plus {
  position: relative;
  width: $space-9x;
  height: $space-9x;
  line-height: $space-7x;
  border-radius: 50%;
  border: 3px solid $color-white;
  composes: caption-lt-lg from 'css/typography.scss';
  text-align: center;
}


.plus-default {
  @extend %plus;
  margin-top: -$space-4x;
  background: $color-caribbean-green;
  color: $color-white;
}

.plus-profile {
  @extend %plus;
  left: -15px;
  font-size: 0.8rem;
  margin-top: 0;
  background: $color-athens-gray-dark;
  color: $color-regent;
}
```

and got the error message `composition is only allowed when selector is single :local class name not in "%plus"`

Reply:
Look at how to use SCSS `@mixins` in `MemberProfile.scss`, that would be one way to handle that. 

Otherwise, better, you can do this:

```
.plus { 
  position: relative;
  width: $space-9x;
  height: $space-9x;
  line-height: $space-7x;
  border-radius: 50%;
  border: 3px solid $color-white;
  text-align: center;
  // default props overriden below
  margin-top: -$space-4x;
  background: $color-caribbean-green;
  color: $color-white;
  &-profile { // makes .plus-profile
    left: -15px;
    font-size: 0.8rem;
    margin-top: 0;
    background: $color-athens-gray-dark;
    color: $color-regent;
  } 
}
```
You can then apply `plus` or `plus plus-profile`, respectively (or whatever sent-in classname applied).

Reply:
Ok, I have updated `RoundImageRow.scss` to use mixins